python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -623,7 +625,7 @@ def __set__(self, obj, _val):
         ## Does not modify val?
         self._check_type(obj, final_val)
 
-        GangaObject.__setattr__(raw_obj, attr_name, final_val)
+        raw_obj.__setattr__(attr_name, final_val)
```

Comment:
Probably, I'm not sure how the function maps but I suspect that should work I'll try the change